models/engine/db_storage.py

```diff
@@ -74,3 +74,21 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Return an object with a given class and ID"""
+        ret = None
+        objects = self.all(cls)
+        lent = len(cls.__name__) + 1
+        for i in objects:
+            if i[lent:] == id:
+                ret = objects[i]
+        return (ret)
+
+    def count(self, cls=None):
+        """Count the amount of objects of a class or all objects"""
+        ret = 0
+        objects = self.all(cls)
+        for i in objects:
+            ret += 1
```

Comment:
you can replace the lines 91-93 for: `ret = len(self.all(cls))`

Reply:
Oh that's clever! A simple but more efficient way of cycling through all instances!